test/gtest/ucs/test_bitmap.cc

```diff
@@ -11,15 +11,15 @@ class test_ucs_bitmap : public ucs::test {
 public:
     virtual void init()
     {
-        UCS_BITMAP_CLEAR(&bitmap);
+        UCS_BITMAP_CLEAR(bitmap);
```

Comment:
it's more clear if any macro/function which changes the bitmap will accept it by pointer
e.g 
UCS_BITMAP_CLEAR(&bitmap);
UCS_BITMAP_INPLACE_NOT(&btimap)